app/src/main/res/values-night/colors.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="onAccent">#5A0933</color> <!--Pink1000-->
```

Comment:
I can't find this one in the material design colors. Looks somewhere in-between pink 800 and pink 900. I tried it locally with pink900 and I think it looked fine so I would recommend switching to that.